src/get-codemods.js

```diff
@@ -1,12 +1,24 @@
 'use strict';
 
 const https = require('https');
+const HttpProxyAgent = require('https-proxy-agent');
 
 module.exports = async function getCodemods(url) {
   let manifest = '';
 
   await new Promise((resolve, reject) => {
-    https.get(url, res => {
+
+    let proxy = process.env.https_proxy
```

Comment:
Is this a convention of some sort?

Reply:
Yes, https_proxy et http_proxy are common use. You can see examples in tests of "node-https-agent" here : 
https://github.com/TooTallNate/node-https-proxy-agent/blob/master/test/test.js

Reply:
Does something missing ?

Reply:
nope, looks good